frontend/packages/console-dynamic-plugin-sdk/src/app/components/index.ts

```diff
@@ -7,3 +7,4 @@ export { default as ResourceStatus } from './utils/resource-status';
 
 export * from './status/icons';
 export * from './status/statuses';
+export { default as ListPageBody } from './factory/ListPage/ListPageBody';
```

Comment:
@fhlavac I know this wasn't caught by the linter, but can we put this export above the wildcard `*` exports?